### ISSUE TITLE
No focus indicator for the toolbar button - accessibility

### ISSUE THREAD

**Issue by sarbbottam** [quilljs/quill#1020]
Focus indicator is extremely important to keyboard only users. Currently the default browser focus indicator is suppressed via [`outline: none` at quill/develop/assets/base.styl](https://github.com/quilljs/quill/blob/develop/assets/base.styl#L25)

![focus-indicator](https://cloud.githubusercontent.com/assets/949380/19006599/55ae6d32-8715-11e6-9f81-19072188dcb6.png)

Either enable browser default focus indicator or compensate it with custom style. You can read more about focus indicator and its necessity at [Keyboard Accessibility and Lost Focus](https://sarbbottam.github.io/accessibility,/css/2016/09/29/keyboard-accessibility-and-lost-focus/)

**Steps for Reproduction**

1. Visit http://quilljs.com/playground/
2. Press `tab` to reach the toolbar buttons

**Expected behavior**:
Any focused element must be visible and identifiable

**Actual behavior**:
Notice that it is not possible to identify which button is focused.

**Platforms**:
Chrome latest, Mac latest

**Version**:
1.0.6

**Comment by sarbbottam** [comment]
Thanks @jhchen.
